revocation/result/results.go

```diff
@@ -116,6 +120,6 @@ type CertRevocationResult struct {
 
 	// RevocationMethod is the method used to check the revocation status of the
 	// certificate, including Unknown(0), MethodOCSP(1), MethodCRL(2) and
-	// OCSPFallbackCRL(3)
-	RevocationMethod int
+	// MethodOCSPFallbackCRL(3)
```

Comment:
Numbers should be removed as it is an enum. BTW, should it be `MethodUnknown`?

Reply:
Updated.